deploy/internal/statefulset-core.yaml

```
@@ -95,6 +95,8 @@ spec:
                 configMapKeyRef:
                   name: noobaa-config
                   key: NOOBAA_LOG_LEVEL
+            - name: CONFIG_JS_RESTRICT_RESOURCE_DELETION
```

An env variable should not start with `CONFIG_JS_`
```suggestion
            - name: RESTRICT_RESOURCE_DELETION
```